server.js

```diff
@@ -0,0 +1,12 @@
+const express = require("express")
+
+const app = express()
```

Comment:
Consider adding error handling middleware to handle any errors that might occur in your application. This can be done by adding a middleware function at the end of your middleware stack that takes four arguments (err, req, res, next). [medium]